hardware/i2c.md

```diff
@@ -49,4 +49,4 @@ No difference was seen between the revisions.
 
 ## Notes
 
-The <span class="v2">V2</span> device can be woken by activating (by pulling low) the combined sensor interrupt on P0.25. This open drain signal is connected between the nRF52, the KL27, and motion sensors and requires the nRF52 internal pull up to be configured, even while the device is sleeping.
+The <span class="v2">V2</span> device can be woken by activating (by pulling low) the combined sensor interrupt on P0.25. This open drain signal is connected between the Target MCU, the Interface MCU, and motion sensors and requires the target MCU internal pull up to be configured, even while the device is sleeping.
```

Comment:
```suggestion
The <span class="v2">V2</span> device can be woken by activating (by pulling low) the combined sensor interrupt on P0.25. This open drain signal is connected between the target MCU, the Interface MCU, and motion sensors and requires the target MCU internal pull up to be configured, even while the device is sleeping.
```

Reply:
Edited this in the original commit to keep the git history clean for merging (if possible I'd like to keep the 3 commits, as they are more or less independent).

Diff between rebases:
https://github.com/microbit-foundation/dev-docs/compare/2747f8e1a8f599e124bbafd1a4b0fafbabacc072..b96c9a28afef09f7aee657bd4f43ec641afa0d3a